ro_help/hub/views.py

```diff
@@ -196,8 +196,9 @@ def get_context_data(self, **kwargs):
         context["current_city"] = self.request.GET.get("city")
         context["current_urgency"] = self.request.GET.get("urgency")
         context["current_search"] = self.request.GET.get("q", "")
-
+        context["current_tags"] = self.request.GET.getlist("tag", "")
         context["counties"] = needs.order_by("county").values_list("county", flat=True).distinct("county")
+        context["tags"] = set(needs.values_list("resource_tags__name", flat=True))
```

Comment:
As a small suggestion, can we order them?

Reply:
Yup, we can.